### ISSUE TITLE
Dockerfile.mingw build fails on sqlite archive

### ISSUE THREAD

**Issue by voltagex** [tatsuhiro-t/aria2#435]
```
Removing intermediate container 60b1d11c8aaa
Step 15 : RUN tar xf sqlite-autoconf-3080700.tar.gz
 ---> Running in 5fb2a9cc9216
tar: sqlite-autoconf-3080700.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
2015/07/18 19:27:41 The command [/bin/sh -c tar xf sqlite-autoconf-3080700.tar.gz] returned a non-zero code: 2
```

**Comment by voltagex** [comment]
It's just a filename fix - pull coming soon.

**Comment by voltagex** [comment]
Have to run out, but here's the patch.


```diff
diff --git a/Dockerfile.mingw b/Dockerfile.mingw
index d21c15d..8012925 100644
--- a/Dockerfile.mingw
+++ b/Dockerfile.mingw
@@ -53,8 +53,8 @@ RUN cd expat-2.1.0 && \
     --build=`dpkg-architecture -qDEB_BUILD_GNU_TYPE` && \
     make install

-RUN tar xf sqlite-autoconf-3080700.tar.gz
-RUN cd sqlite-autoconf-3080700 && \
+RUN tar xf sqlite-autoconf-3080803.tar.gz
+RUN cd sqlite-autoconf-3080803 && \
     ./configure \
     --disable-shared \
     --enable-static \

```

**Comment by tatsuhiro-t** [comment]
Thank you.  Your patch was merged in master via 4b924702bc740c3cd8f64caa48ff815ebb0a8924
